### PR TITLE
fix: Skip dangerous tests while not in CI

### PR DESCRIPTION
**Pull request by Xuanwo**
Signed-off-by: Xuanwo <github@xuanwo.io>

Related to https://github.com/aliyun/aliyun-cli/issues/568

### CODE REVIEW

File: cli/completion_installer_test.go [hunk 1]
```diff
@@ -74,31 +71,32 @@ func TestZshInstaller(t *testing.T) {
...
 }
 
-func TestCompletionInstallers(t *testing.T) {
-	i := completionInstallers()
-	if runtime.GOOS == "windows" {
-		assert.Empty(t, i)
-	} else {
-		assert.NotNil(t, i)
-	}
+// func TestCompletionInstallers(t *testing.T) {
```
Comment (choleraehyq) [inline cli/completion_installer_test.go:74]: t.SkipNow()  就行了，用不着注释

**Comment by JacksonTian** [comment]
在 CI 环境应该允许执行。

File: cli/completion_installer_test.go (hunk 1 above)
Comment (Xuanwo) [reply cli/completion_installer_test.go]: Thanks!

**Comment by Xuanwo** [comment]
> 在 CI 环境应该允许执行。

Implemented in https://github.com/aliyun/aliyun-cli/pull/569/commits/71382a763deb3dfa3fc97d34b549e118084e6a98

**Comment by Xuanwo** [comment]
> Implemented in [71382a7](https://github.com/aliyun/aliyun-cli/commit/71382a763deb3dfa3fc97d34b549e118084e6a98)

Oh, CI is not running on github action. Sorry for that, fix is coming.

**Review by JacksonTian — APPROVED** [review]

**Pull request merged**